pom.xml

```diff
@@ -53,7 +53,7 @@
         <dep.drift.version>1.14</dep.drift.version>
         <dep.joda.version>2.10.3</dep.joda.version>
         <dep.tempto.version>166</dep.tempto.version>
-        <dep.testng.version>6.10</dep.testng.version>
+        <dep.testng.version>6.14.3</dep.testng.version>
```

Comment:
What good does it bring?

Reply:
Not sure, testng release notes are pretty vague https://github.com/cbeust/testng/blob/master/CHANGES.txt. 

There are couple bug fixes, and maybe new bugs as well.